chalice/build_deploy_config.sh

```diff
@@ -52,7 +52,8 @@ cat "$config_json" | jq ".stages.$stage.tags.DSS_DEPLOY_ORIGIN=\"$DEPLOY_ORIGIN\
 	.stages.$stage.tags.service=\"${DSS_INFRA_TAG_SERVICE}\"  | \
 	.stages.$stage.tags.project=\"$DSS_INFRA_TAG_PROJECT\" | \
 	.stages.$stage.tags.owner=\"${DSS_INFRA_TAG_OWNER}\" | \
-	.stages.$stage.tags.env=\"${DSS_DEPLOYMENT_STAGE}\""  | sponge "$config_json"
+	.stages.$stage.tags.env=\"${DSS_DEPLOYMENT_STAGE}\" | \
+	.stages.$stage.app_name=\"env.lambda_name\"" | sponge "$config_json"
```

Comment:
Should this be changed to include the stage name too? Like, `env.lambda_name-env.stage-name`?

Reply:
the `env.lambda_name` has the stage name in it (ln. 17) but good eyes 🦉